syntax = "proto3";
package fixbug;

//import "comm.proto";
import "src/protobuf/comm.proto";

option cc_generic_services = true;


message GetFriendListRequest
{
  uint32 id = 1;
}

message GetFriendListResponse
{
  ResultCode result = 1;
  repeated bytes friends = 2;
}


// 好友模块
service FriendServiceRpc
{
rpc GetFriendList(GetFriendListRequest) returns (GetFriendListResponse);
}
